src/libraries/System.Transactions.Local/tests/OleTxTests.cs

```diff
@@ -528,25 +528,26 @@ public void ImplicitDistributedTransactions_cannot_be_changed_after_being_set()
         });
     }
 
-    [ConditionalFact(nameof(IsRemoteExecutorSupportedAndNotNano))]
-    public void ImplicitDistributedTransactions_cannot_be_changed_after_being_read_as_true()
-    {
-        // Temporarily skip on 32-bit where we have an issue.
-        if (!Environment.Is64BitProcess)
-        {
-            return;
-        }
-
-        using var _ = RemoteExecutor.Invoke(() =>
-        {
-            TransactionManager.ImplicitDistributedTransactions = true;
-
-            Test(MinimalOleTxScenario);
-
-            Assert.Throws<InvalidOperationException>(() => TransactionManager.ImplicitDistributedTransactions = false);
-            TransactionManager.ImplicitDistributedTransactions = true;
-        });
-    }
+// Disabled as per https://github.com/dotnet/runtime/issues/77241#issuecomment-1317114960
+//    [ConditionalFact(nameof(IsRemoteExecutorSupportedAndNotNano))]
+//    public void ImplicitDistributedTransactions_cannot_be_changed_after_being_read_as_true()
+//    {
+//        // Temporarily skip on 32-bit where we have an issue.
+//        if (!Environment.Is64BitProcess)
+//        {
+//            return;
+//        }
+//
+//        using var _ = RemoteExecutor.Invoke(() =>
+//        {
+//            TransactionManager.ImplicitDistributedTransactions = true;
+//
+//            Test(MinimalOleTxScenario);
+//
+//            Assert.Throws<InvalidOperationException>(() => TransactionManager.ImplicitDistributedTransactions = false);
+//            TransactionManager.ImplicitDistributedTransactions = true;
+//        });
+//    }
```

Comment:
Can you just add `[ActiveIssue("https://github.com/dotnet/runtime/issues/77241")]` to it instead?

Reply:
Thanks, I was looking for the way to do this in runtime...

Reply:
(assuming it's OK to add this alongside [ConditionalFact]....)

Reply:
Yup